packages/deployment/upgrade-test/upgrade-test-scripts/start_to_to.sh

```diff
@@ -32,6 +32,13 @@ if ! test -f "$HOME/.agoric/runActions-${THIS_NAME}"; then
   touch "$HOME/.agoric/runActions-${THIS_NAME}"
 fi
 
+#region alkjdf
```

Comment:
didn't mean to push this. disregard
(draft but I'll clean up before real review)

Reply:
This is probably the reason tests are failing.  Can you please remove it... I prefer not to be pinged for review of something with failing tests.

Reply:
apologies. I was hoping you could speak to the questions in the comments